### PR TITLE
Update readme & create publication overview

### PR DESCRIPTION
**Pull request by sarah-hlsn**
SCRUM [user story #125](https://tree.taiga.io/project/emanuel-schmid-climada/us/125?milestone=395053)

This pull request adjusts the information provided in the climada_papers repository, based on the new policy for making zenodo releases. Changes include:

New: `climada_papers_overview.md`
- move paper overview from README to new file & include zenodo DOI links in overview

Adjustments: `README.md `
- explain repository
- point external users to the overview 
- give internal users guidance for how to publish code and add to the overview

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,65 +1,11 @@
-# climada_papers
+# READ ME
+This repository contains an overview of the publications an corresponding software components implemented in [CLIMADA Python](https://github.com/CLIMADA-project/climada_python). Please see [climada_papers_overview.md](./climada_papers_overview.md) for a list of publications and Zenodo DOI links.
```
Comment (luseverin) [inline README.md]:
```suggestion
This repository contains an overview of the publications and corresponding software components implemented in [CLIMADA Python](https://github.com/CLIMADA-project/climada_python). Please see [climada_papers_overview.md](./climada_papers_overview.md) for a list of publications and Zenodo DOI links.
```

File: README.md [hunk 2]
```diff
@@ -1,65 +1,11 @@
...
-Aznar-Siguan, G. and Bresch, D. N., 2019: CLIMADA v1: a global weather and climate risk assessment platform, Geosci. Model Dev., https://doi.org/10.5194/gmd-12-3085-2019
-
-## Forthcoming papers & publications in preparation:
-
-Please check www.wcr.ethz.ch
+* Create your own GitHub repository to store the code
+* Upload all relevant code
+* Add a readme file with all the information necessary to replicate your analysis, including which CLIMADA version was used
+* Make a Zenodo release
+* Update the paper overview in the climada_papers repository with the paper DOI and the code DOI from Zenodo (create a new row on top of the existing table)
```
Comment (luseverin) [inline README.md]:
```suggestion
* Update the paper overview [climada_papers_overview.md](./climada_papers_overview.md) in the climada_papers repository with the paper DOI and the code DOI from Zenodo (create a new row on top of the existing table)
```

File: climada_papers_overview.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# CLIMADA-papers overview
+This file contains an overview of the publications an corresponding software components implemented in [CLIMADA Python](https://github.com/CLIMADA-project/climada_python).
```
Comment (luseverin) [inline climada_papers_overview.md]:
```suggestion
This file contains an overview of the publications and corresponding software components implemented in [CLIMADA Python](https://github.com/CLIMADA-project/climada_python). 
```

File: climada_papers_overview.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# CLIMADA-papers overview
+This file contains an overview of the publications an corresponding software components implemented in [CLIMADA Python](https://github.com/CLIMADA-project/climada_python). 
+
+For authors: Please consult the [README](./README.md) on how to add to this overview before making any edits.
```
Comment (luseverin) [inline climada_papers_overview.md]: Maybe "to authors" or "to contributors" instead? The "for authors" sounds a bit weird to me but maybe that's just me. (also not so important, feel free to disregard)

**Review by luseverin — CHANGES_REQUESTED** [review]
Thanks for the work! I made a few suggestions (typos and suggestions for rephrasing). Otherwise it looks all good to me.

File: climada_papers_overview.md (hunk 4 above)
Comment (sarah-hlsn) [reply climada_papers_overview.md]: Yes I that's a good point, will change it accordingly :)

**Review by spjuhel — APPROVED** [review]

**Pull request merged**